build-ios-shared.sh

```
@@ -169,136 +161,67 @@ lipo \
-  dst=$d/sherpa_onnx.framework
+  cat > $fw_dir/Modules/module.modulemap << 'MEOF'
+framework module SherpaOnnxC {
+  header "sherpa-onnx/c-api/c-api.h"
+  export *
+}
+MEOF
 
-  # The Info.plist is modified from
-  # https://github.com/Spicely/flutter_openim_sdk_ffi/blob/main/ios/openim_sdk_ffi.framework/Info.plist
-  cat >$dst/Info.plist <<EOF
+  cat > $fw_dir/Info.plist << 'PEOF'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)